spec/migrations/20210718074605_update_ibm_vendor_to_ibm_cloud_spec.rb

```diff
@@ -0,0 +1,65 @@
+require_migration
+
+RSpec.describe UpdateIbmVendorToIbmCloud do
+  let(:vm_stub) { migration_stub(:VmOrTemplate) }
+
+  migration_context :up do
+    it "updates vendor to 'ibm_cloud' for IBM Cloud managers" do
+      vpc_vm = vm_stub.create!(:type => "ManageIQ::Providers::IbmCloud::VPC::CloudManager::Vm", :vendor => "ibm")
+      vpc_template = vm_stub.create!(:type => "ManageIQ::Providers::IbmCloud::VPC::CloudManager::Template", :vendor => "ibm")
+
+      powervs_vm = vm_stub.create!(:type => "ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::Vm", :vendor => "ibm")
+      powervs_template = vm_stub.create!(:type => "ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::Template", :vendor => "ibm")
+
+      powervc_vm = vm_stub.create!(:type => "ManageIQ::Providers::IbmPowerVc::CloudManager::Vm", :vendor => "ibm_power_vc")
+      powervc_template = vm_stub.create!(:type => "ManageIQ::Providers::IbmPowerVc::CloudManager::Template", :vendor => "ibm_power_vc")
+
+      openstack_vm = vm_stub.create!(:type => "ManageIQ::Providers::Openstack::CloudManager::Vm", :vendor => "ibm")
+      openstack_template = vm_stub.create!(:type => "ManageIQ::Providers::Openstack::CloudManager::Template", :vendor => "ibm")
```

Comment:
Minor (doesn't impact the test but looks strange) the the `:vendor` for these two would be `"openstack"`

Reply:
Fixed!